mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -21,230 +29,280 @@ instances:
   - server: localhost
 
     ## @param user - string - required
-    ## Datadog Username created to connect to MySQL.
+    ## Datadog username created to connect to MySQL.
     #
-    user: datadog
+    user: <USER>
```

Comment:
(nit) Should we keep it to `datadog` to suggest using a specific user for monitoring?

If we use `<USER>`, the the descriptions are currently not suitable I think and might be improved:

- `Datadog username created to connect to MySQL.`
- `Password associated to the Datadog user.`

e.g.

- `User used to connect to MySQL.`
- `Password associated to the MySQL user.`